core/o11y/src/lib.rs

```diff
@@ -110,17 +114,27 @@ pub struct DefaultSubscriberGuard<S> {
 }
 
 // Doesn't define WARN and ERROR, because the highest verbosity of spans is INFO.
-#[derive(Copy, Clone, Debug, Default, clap::ValueEnum, serde::Serialize, serde::Deserialize)]
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    Default,
+    clap::ValueEnum,
+    serde::Serialize,
+    serde::Deserialize,
+    strum::AsRefStr,
+)]
 pub enum OpenTelemetryLevel {
     #[default]
     OFF,
+    ERROR,
+    WARN,
```

Comment:
Adding these variants makes the comment above `OpenTelemetryLevel` become incorrect. It not having `error` or `warn` as variants IIRC was serving the purpose of those levels not being specifyable where you can filter by level…?